playwright.config.js

```diff
@@ -17,11 +17,11 @@ module.exports = defineConfig({
   /* Fail the build on CI if you accidentally left test.only in the source code. */
   forbidOnly: !!process.env.CI,
   /* Retry on CI only */
-  retries: process.env.CI ? 2 : 0,
+  retries: 3,
```

Comment:
Does this mean we always retry 3 times? Do we still need the comment above this line?

Reply:
Yeah. We retry if failed thrice both locally and on CI.